.github/workflows/main.yaml

```diff
@@ -55,13 +55,13 @@ jobs:
         if: ${{ github.event_name == 'push' }}
         with:
           name: tested-charm
-          path: .tox/**/namecheap-acme-operator_ubuntu-20.04-amd64.charm
+          path: .tox/**/namecheap-acme-operator_ubuntu-22.04-amd64.charm
```

Comment:
I remember this failing when I was working on it, because some dependencies from the tls lib won't be installed correctly, I have checked the last run of the CI for this PR, and the integration test ran on ubuntu20.04, because I see that part wasn't changed.
I expect to fail if we change that.
```
integration-test:
    name: Integration tests
    runs-on: ubuntu-22.04
```

Reply:
Done